web/views.py

```
@@ -6772,3 +6774,257 @@ def delete_post(request, post_id):
+
+    if request.method == "POST":
+        # Handle creation of a new map
+        title = request.POST.get("title")
+        description = request.POST.get("description", "")
+        public = request.POST.get("public") == "on"
+
+        if title:
+            new_map = LearningMap.objects.create(user=request.user, title=title, description=description, public=public)
+            messages.success(request, "Learning map created successfully!")
+            return redirect("learning_map_detail", slug=new_map.slug)
+        else:
```

Ruff (0.8.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)